doc/source/metadata/Introduction.rst

```diff
@@ -0,0 +1,724 @@
+    ``(comment: of air_temperature)``, 
+    as they now refer to the main variable that is a maximum over over 12 hours. 
+
+Percentile coordinate variable
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As the main variable provides the air temperature at a set of percentiles,
+a further dimensional coordinate variable is required to allow the data to be 
+fully interpreted. This holds the set of percentile values. 
+
+.. literalinclude:: temp12max_perc_ncdump.txt
+    :tab-width: 4
+    :lines: 24-26, 79-82
+
+The variable attributes are:
+
+long_name
+   ``percentile`` which is used all IMPROVER percentile variables, 
+   is not in the `CF Standard Name`_ list.
+
+units
+    The units here are a percentage. 
+
+
+Spot data 
+*********
+
+Broadly, the spot data parallels the form of the gridded data, 
+both probabilities and percentiles are produced,
+but there are some notable differences because of the nature of data.
+These difference are described below, but the full ncdump listing
+is shown in an appendix.
+
+Global attributes 
+^^^^^^^^^^^^^^^^^
+
+Only the title is different for the spot data. 
+
+.. literalinclude:: temp12max_spotperc_ncdump.txt
+    :tab-width: 4
+    :lines: 60-73
+    :emphasize-lines: 13
+
+Dimensions
+^^^^^^^^^^
+
+The dimensions for the horizontal coordinate variables have been replaced by
+a dimension for an index coordinate for the sites, 
+and a dimension for the WMO identifier string length has been added. 
+
+.. literalinclude:: temp12max_spotperc_ncdump.txt
+    :tab-width: 4
+    :lines: 24-27
+
+spot_index
+    The dimension for the index for the set of sites. 
+
+string5 / string8
+    Just constants used to dimension the character length of the string variable
+    holding WMO identifier and Met Office identifiers, respectively.
+
+.. how tied to the code are these identifiers, as they may not be appropiate
+    for other users?
+
+
+Variables
+^^^^^^^^^
+
+The majority of variables are the same for both gridded data and spot data.
+Only the differences will be discussed here.
+The variable attributes are common to the gridded data, so these will not be discussed. 
+
+.. literalinclude:: temp12max_spotperc_ncdump.txt
+    :tab-width: 4
+    :lines: 18-23, 43-50, 57-58
+
+spot_index
+    An arbitrary integer index for the sites. 
+
+altitude
+    Height of site above sea level. 
+
+latitude, longitude
+    Site positions are only provided in latitude and longitude,
+    regardless of the projection used for the corresponding grid.
+    These can be considered as relative the WGS84 or the World Geodetic System 1984 datum,
+    although this is not explicit in the metadata. 
+
+wmo_id
+    For WMO sites, this is the numerical WMO identifier stored as a string; 
+    for other sites this is a set to ``NaN`` (at present, the formatting of this is not ideal, but will be changed to ensure that the numbers are right-justified and padded with zeros in the next release). 
+
```

Comment:
Needs a section on the `met_office_site_id`. It is worth highlighting here that the `met_office_site_id` name is user configurable, such that it can be changed for different institutions / indices. It is a zero-padded string.

Reply:
Added. Thanks.